Library/Homebrew/manpages/brew.1.md

```diff
@@ -574,6 +574,14 @@ can take several different forms:
 
     *Note:* Homebrew will only try to print emoji on Lion or newer.
 
+  * HOMEBREW\_NO\_INSECURE\_REDIRECT:
+    If set, Homebrew will not permit redirects from secure download
+    protocols to insecure ones.
```

Comment:
Might want to just say HTTPS/HTTP here as it'd just handling that and not e.g. FTP.